src/data/lib/prim/Agda/Builtin/Nat.agda

```diff
@@ -74,6 +74,8 @@ div-helper k m (suc n) (suc j) = div-helper k       m n j
 
 {-# BUILTIN NATDIVSUCAUX div-helper #-}
 
+{-# COMPILE JS div-helper = k => m => n => j => k + (n + m - j) / (1n + m) #-}
```

Comment:
What does `1n` mean?

Reply:
`1n` is a [`BigInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) literal one in JS.

As I've noticed, Agda JS compiler has been compiling `Nat` into `BigInt` since [#4878](https://github.com/agda/agda/issues/4878).

Reply:
Thanks!